app/routes/public/exhibition/view.js

```diff
@@ -16,10 +17,25 @@ export default class ViewRoute extends Route {
     });
   }
 
-  afterModel(model) {
+  async afterModel(model) {
     super.afterModel(...arguments);
     if (model.exhibitor.status !== 'accepted') {
       this.transitionTo('not-found');
     }
+    const exhibitors = await model.event.query('exhibitors', {
```

Comment:
If you await here, it'll block till loading all exhibitors

Reply:
Ok then what should be done?